exporter/awsxrayexporter/translator/aws.go

```diff
@@ -146,6 +154,13 @@ func makeAws(attributes map[string]string, resource pdata.Resource) (map[string]
 			VersionLabel: awsxray.String(versionLabel),
 		}
 	}
+	if clusterName != "" {
+		eks = &awsxray.EKSMetadata{
```

Comment:
If someone happens to export to xray from a different cloud like GCP, we'll still fill in EKS metadata. I think it's ok better than just dropping the data.